R/adp.R

```diff
@@ -3788,6 +3788,8 @@ display.bytes <- function(b, label="", ...)
 #'
 #' @param x an [adp-class] object that contains bottom-tracking velocities.
 #'
+#' @param despike boolean that indicates if the bottom velocities should be despiked
```

Comment:
1. I think "boolean" should be "logical value" because that's the R way.
2. I think "if" should be "whether".
3. There should be a period at the end of the sentence (matching the `x` arg).
4. I think it there should be a note on how it's done.  So, change "despiked" to "despiked, using [despike()]", which will give a cross-reference that users can get by clicking.